static_src/stores/service_plan_store.js

```diff
@@ -54,7 +54,8 @@ export class ServicePlanStore extends BaseStore {
       case serviceActionTypes.SERVICE_PLAN_RECEIVED: {
         const servicePlan = action.servicePlan;
         const servicePlanReceived = Object.assign({}, servicePlan, { fetching: false });
-        this.merge('guid', servicePlanReceived);
+        this.merge('guid', servicePlanReceived, () => { });
+        this.emitChange();
```

Comment:
You're correct. It should only be necessary when some other state is fetched outside of the merge.